de-lib/src/main/java/org/iplantc/de/diskResource/client/gin/DiskResourceGinModule.java

```diff
@@ -63,7 +63,6 @@
 import org.iplantc.de.diskResource.client.views.sharing.dialogs.ShareResourceLinkDialog;
 import org.iplantc.de.diskResource.client.views.toolbar.DiskResourceViewToolbarImpl;
 import org.iplantc.de.diskResource.client.views.toolbar.FileUploadByUrlViewImpl;
-import org.iplantc.de.diskResource.client.views.toolbar.HTPathListAutomationViewImpl;
```

Comment:
Missed this while rebasing :(

Reply:
Sorry, my refactors getting merged probably caused all of your conflicts ☹️

Reply:
No worries.